### PR TITLE
New package: jump-0.30.1

### PR DESCRIPTION
**Pull request by aweagel**

### CODE REVIEW

**Comment by aweagel** [comment]
Given that I am relatively new to Void (and golang packaging) this PR should be viewed as more of a feature request than anything.

Just a happy `jump` user hoping to get the package included in the official repos :)

That said, perhaps someone who has more experience with `build_style=go` can help me with this one? Maybe @Vaelatern @the-maldridge or @thypon?

File: srcpkgs/jump/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+revision=1
+build_style=go
+go_import_path=github.com/gsamokovarov/jump
+go_package=github.com/gsamokovarov/jump/cmd/jump
+hostmakedepends="git"
+short_desc="Fuzzy directory jumper that learns your habits"
+maintainer="Arthur Weagel <arthur.contrib@gmail.com>"
+license="MIT"
+homepage="https://github.com/gsamokovarov/jump"
+changelog="https://github.com/gsamokovarov/jump/releases"
```
Comment (Vaelatern) [inline srcpkgs/jump/template:13]: Ideally something plaintext

File: srcpkgs/jump/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'jump'
+pkgname=jump
+version=0.30.1
+revision=1
+build_style=go
+go_import_path=github.com/gsamokovarov/jump
+go_package=github.com/gsamokovarov/jump/cmd/jump
```
Comment (Vaelatern) [inline srcpkgs/jump/template:7]: Perhaps part of the problem is this doesn't exist.

Try removing go_package and see what happens?

File: srcpkgs/jump/template (hunk 1 above)
Comment (aweagel) [reply srcpkgs/jump/template:12]: There is no `CHANGELOG` file, so this is the closest option I'm aware of (the author does provide some notes with each release). There's also the [atom feed version](https://github.com/gsamokovarov/jump/releases.atom) for the releases page if that's preferable.

File: srcpkgs/jump/template (hunk 2 above)
Comment (aweagel) [reply srcpkgs/jump/template]: @Vaelatern That did the trick! I've marked this PR as ready for review.

File: srcpkgs/jump/template (hunk 1 above)
Comment (Vaelatern) [reply srcpkgs/jump/template:12]: Better to not have it at all if it doesn't have it.

Comment (aweagel) [reply srcpkgs/jump/template:12]: Hmm, I see this being done in lots of package definitions (see [hub](https://github.com/void-linux/void-packages/blob/master/srcpkgs/hub/template), [erlang](https://github.com/void-linux/void-packages/blob/master/srcpkgs/erlang/template), [mpv](https://github.com/void-linux/void-packages/blob/master/srcpkgs/mpv/template), [obs](https://github.com/void-linux/void-packages/blob/master/srcpkgs/mpv/template), [skaffold](https://github.com/void-linux/void-packages/blob/master/srcpkgs/skaffold/template), [sysdig](https://github.com/void-linux/void-packages/blob/master/srcpkgs/sysdig/template), etc).

Are you sure it's better to remove it, or would it make sense to follow the existing convention? I don't have a strong preference.

Comment (Vaelatern) [reply srcpkgs/jump/template:12]: Well, that wasn't the intention, but if this is where it is, that's where it is. I will not object further.

**Comment by Vaelatern** [comment]
Thank you for your contribution!

**Pull request merged**